pretext/ExtraTopics/BigO.ptx

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0"?>
+<section xml:id="extratopics-bigo">
+  <title>Big O Analysis</title>
+
+  <p>It's impossible to experimentally measure the theoretical performance of algorithms on a computer as the size of our data (which we call n, to some factor) goes to infinity,
+    or gets really big. We won't get deep into it, but this is due to things like variable hardware between computers and just general performance of software.</p>
```

Comment:
I'm not totally sure about this analogy, it may just distract from what we're talking about. I was just trying to give a brief introduction. Greg wanted a more mathematical introduction altogether, but I also didn't want that to distract from what we're doing.

Reply:
I don't love that analogy either - I don't find it to be the most intuitive. Tbh I like the explanation that's there more.

Reply:
Ok, let's skip that then